javascript/react/src/components/StepSelector.hooks.ts

```diff
@@ -30,7 +30,16 @@ const groups = [
 	},
 	{
 		label: 'Combine columns',
-		verbs: ['boolean', 'derive', 'fold', 'unfold', 'onehot', 'merge', 'spread'],
+		verbs: [
+			'boolean',
+			'derive',
+			'fold',
+			'unfold',
+			'onehot',
+			'merge',
+			'spread',
+			'unhot',
```

Comment:
Can you put this right after onehot? I've been trying to pair them up when there is a do/undo option.